src/backend/executor/nodeMaterial.c

```diff
@@ -759,3 +764,30 @@ ExecEagerFreeMaterial(MaterialState *node)
 	}
 }
 
+
```

Comment:
Again, for function that is not ready for other files' use, please add "static". Please also double-check other functions.